src/System Application/Test/Document Processor/src/PDFDocumentProcessorTest.Codeunit.al

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.IO;
+
+using System.Utilities;
+using System.TestLibraries.Utilities;
+
+
+codeunit 132601 "PDF Document Processor Test"
+{
+    Access = Internal;
+    Subtype = Test;
+    TestPermissions = Disabled;
+
+    var
+        Assert: Codeunit "Library Assert";
+        LengthErr: Label 'Outstream length should have length 0';
+
+    [Test]
+    procedure ValidPdfToPngImage()
+    var
+        PdfHelper: Codeunit "PDF Document Processor";
+        TempBlob: Codeunit "Temp Blob";
+        ImageFormat: Enum "Image Format";
+        PdfInstream, ImageStream : InStream;
+    begin
+        // Setup
+        NavApp.GetResource('test.pdf', PdfInstream, TextEncoding::UTF8);
+        TempBlob.CreateInStream(ImageStream);
+        PdfHelper.ConvertToImage(PdfInstream, ImageStream, ImageFormat::Png, 1);
+        Assert.AreNotEqual(0, TempBlob.Length(), LengthErr);
```

Comment:
Please improve the test with check that the proper image is returned. You can compare 2 base64 strings.